project/.travis/install_common.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+set -ev
+
+composer update --prefer-dist --no-interaction --prefer-stable ${COMPOSER_FLAGS}
```

Comment:
AFAIK, it will update the project dependency. What is the point with external tools?

It does not seems very clear IMHO.

Reply:
The `vendor-bin` plugin uses the normal composer build and registers a post update hook which will then load the composer-bin dependencies, so we need a normal composer install/update first.

https://github.com/bamarni/composer-bin-plugin

Reply:
You need the `bin` suffix:

```
composer bin update
```

https://github.com/bamarni/composer-bin-plugin#usage

Am I right?

Reply:
Yes, but you need at least the plugin itself, which is installed by the *normal* composer